src/envelope.js

```diff
@@ -128,9 +128,15 @@ define(function (require) {
     p5sound.soundArray.push(this);
   };
 
+  // THIS DOESN'T WORK YET
+  p5.Env = function(t1, l1, t2, l2, t3, l3) {
+    console.warn('WARNING: p5.Env is now deprecated and may be removed in future versions. Please use the new p5.Envelope instead.');
+    p5.Envelope(t1, l1, t2, l2, t3, l3);
```

Comment:
I would think of p5.Env as extending the p5.Envelope similar to the way that these named p5.SqrOsc extends p5.Oscillator https://github.com/processing/p5.js-sound/blob/master/src/oscillator.js#L566 using `call` and Object.create to extend the prototype. In es6 this can be done with `class` and `extends` but we have been using the es5 approach.

You could use the arguments object or the es6 version, using rest parameters + spread operator, when calling

```    p5.Envelope.call(this, ...args);```